### PR TITLE
is_complex

### PR DESCRIPTION
**Pull request by HadeerArafa108**
Close https://github.com/unifyai/ivy/issues/9903

### CODE REVIEW

**Comment by Daniel4078** [comment]
Can you post the link to the issue that this pull request is dealing with?

**Comment by HadeerArafa108** [comment]
https://github.com/unifyai/ivy/issues/7

**Comment by Daniel4078** [comment]
well you need to create a issue named "is_complex"and reference in the TODO list issue to mark this task as currently being dealt with, then you link this pull request with the "is_complex" issue you just create, so closing this pull request will also close that issue and update the info on the coorisponding TODO list.

**Comment by HadeerArafa108** [comment]
Close #9903

**Review by Daniel4078 — CHANGES_REQUESTED** [review]
Please try to remove your changes in your commits to the files in the .idea and runConfigurations folder, as the content of these files will be shared with other pull requests and should not be altered in usual situations.

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor_functions.py [hunk 1]
```diff
@@ -104,3 +104,30 @@ def test_torch_is_nonzero(
...
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=x[0],
+    )
```
Comment (Daniel4078) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_tensor_functions.py:133]: please do not forget to keep a blank line at the end of the file

**Comment by HadeerArafa108** [comment]
i am so sorry for the trouble I just added the blank line and restored the .idea files

**Comment by Daniel4078** [comment]
after you dealt with the last ivy.iml file, I will quick do a final check on your code and then merge it for you. Thank you!

**Comment by HadeerArafa108** [comment]
thank you so much for the help

**Pull request merged**